test/recipes/70-test_dtls13epoch.t

```diff
@@ -0,0 +1,117 @@
+#! /usr/bin/env perl
+# Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use strict;
+use feature 'state';
+
+use OpenSSL::Test qw/:DEFAULT cmdstr srctop_file bldtop_dir/;
+use OpenSSL::Test::Utils;
+use TLSProxy::Proxy;
+use TLSProxy::Message;
+
+my $test_name = "test_dtls13epoch";
+setup($test_name);
+
+plan skip_all => "DTLSProxy isn't usable on $^O"
+    if ($^O =~ /^(VMS)$/) || ($^O =~ /^(MSWin32)$/);
+
+plan skip_all => "$test_name needs the dynamic engine feature enabled"
+    if disabled("engine") || disabled("dynamic-engine");
+
+plan skip_all => "$test_name needs the sock feature enabled"
+    if disabled("sock");
+
+plan skip_all => "DTLSProxy does not support partial messages"
+    if disabled("ec");
```

Comment:
I'm failing to see the connection between the "partial messages" error and the check for "ec" being enabled

Reply:
I don't either TBH. But somehow we end up in

https://github.com/openssl/openssl/blob/d85ea406479f972ee5b9e832d360543604435200/util/perl/TLSProxy/Message.pm#L220C17-L235C35

I'd like to fix the functionality but I see it as a separate issue

Reply:
It could be due to longer length of some messages with DH instead of ECDH. @mattcaswell could you please approve this as is?

Reply:
Could you add a TODO to investigate this at some later point and then I can approved?

Reply:
Done. Do you also want the same TODO text for other tests where I've done a similar skip of tests?